test/integration/puppetlabs/services/jruby/jruby_metrics_service_test.clj

```diff
@@ -202,6 +203,19 @@
    :current-metrics-values IFn
    :jruby-service (schema/protocol jruby-protocol/JRubyPuppetService)})
 
+(defn expected-values-atom
+  []
+  (let [num-jrubies (if jruby-puppet-core/multithreaded? 1 2)]
+    (atom {:num-jrubies num-jrubies
+           :num-free-jrubies num-jrubies
+           :requested-count 0
+           :borrow-count 0
+           :borrow-timeout-count 0
+           :borrow-retry-count 0
+           :return-count 0
+           :current-requested-instances 0
+           :current-borrowed-instances 0})))
+
```

Comment:
Since this is only used in one spot, can we leave it in-line like the others are?

Reply:
sure